guides/plug.md

```diff
@@ -301,3 +301,6 @@ To make this all work, we converted the nested blocks of code and used `halt(con
 At the end of the day, by replacing the nested blocks of code with a flattened series of plug transformations, we are able to achieve the same functionality in a much more composable, clear, and reusable way.
 
 To learn more about plugs, see the documentation for the [Plug project](https://hexdocs.pm/plug), which provides many built-in plugs and functionalities.
+
```

Comment:
Nitpicking, I would add two empty lines since it is the end of the page and what follows is just reference links

Reply:
Good point :+1: